docs/source/installation.rst

```diff
@@ -283,6 +283,17 @@ Then you need to set the following environment variables:
 
 Finally you can build the package using the standard command.
 
+Release versions - troubleshooting on Mac OSX
```

Comment:
are you sure this is for release versions (which you install from `pip` or `conda` and not just for development versions which you install from a local clone?

I think you mean development version, since you actually install `pip -e`. Right?